common/conf/conf.cpp

```diff
@@ -203,6 +214,11 @@ expected::ExpectedSize MenderConfig::ProcessCmdlineArgs(
 		}
 	}
 
+	if (help_arg) {
+		cli::PrintCliHelp(app);
+		return expected::unexpected(error::MakeError(error::ExitWithSuccessError, ""));
```

Comment:
No definite answer obvs, but should we error on no args?

Reply:
I don't understand. You mea in combination with `--help` or in general? In general it will fail later with something like "an action needs to be specified".

From the options parsed here, is only `--help` and `--version` that are allowed with no command, the rest will require a command.

Also, note that `--version` does not accept other options or the command while `--help` accepts them but ignores. This is based on personal taste and I am open to make both same strict.

Reply:
No, I was thinking of the `exit code` when no arguments are provided :)

Reply:
> In general it will fail later with something like "an action needs to be specified".

Roger, let's keep it as it is then :)